frontend/pages/beta/index.tsx

```diff
@@ -0,0 +1,161 @@
+import useDebounce from '../../utils/hooks/useDebounce'
+
+export default function ExploreModels() {
+  const [group, setGroup] = useState<ListModelType>('all')
+  // TODO - fetch model tags from API
+  const [filter, setFilter] = useState('')
+  const debouncedFilter = useDebounce(filter, 250)
+
+  const { models, isModelsError, mutateModels } = useListModels(group, debouncedFilter)
+
+  const theme = useTheme()
+
+  const error = MultipleErrorWrapper(`Unable to load marketplace page`, {
+    isModelsError,
+  })
+  if (error) return error
+
+  const handleFilterChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setFilter(e.target.value)
+  }
+
+  const onFilterSubmit = (e: React.FormEvent) => {
+    e.preventDefault()
+  }
+
+  const updateSelectedTasks = (selectedTags: string) => {
+    // TODO - When tasks are selected, filter the array of models based on the selection
+    console.log(selectedTags)
+  }
+
+  const updateSelectedLibraries = (selectedTags: string) => {
+    // TODO - When libraries are selected, filter the array of models based on the selection
+    console.log(selectedTags)
+  }
+
+  const updateSelectedType = (selectedType: string) => {
+    if (selectedType) {
+      switch (selectedType) {
+        case MarketPlaceModelSelectType.MY_MODELS:
+          setGroup(MarketPlaceModelGroup.MY_MODELS)
+          break
+        case MarketPlaceModelSelectType.FAVOURITES:
+          setGroup(MarketPlaceModelGroup.FAVOURITES)
+          break
+        default:
+          setGroup(MarketPlaceModelGroup.ALL)
+      }
+      mutateModels()
+    }
+  }
+
+  return (
+    <Wrapper title='Explore Models' page='marketplace'>
+      <TextField
+        color='primary'
+        onClick={onFilterSubmit}
+        sx={{
+          maxWidth: '400px',
+          marginBottom: 3,
+        }}
+        placeholder='Filter Models'
+        value={filter}
+        onChange={handleFilterChange}
+        InputProps={{
+          startAdornment: (
+            <InputAdornment position='start'>
+              <SearchIcon />
+            </InputAdornment>
+          ),
+        }}
+      />
+      <Grid container>
+        <Grid item sm={8} xs={12}>
+          <Paper sx={{ py: 2, px: 4 }}>
+            <Box sx={{ mb: 2, borderBottom: 1, borderColor: 'divider' }} data-test='indexPageTabs'>
+              <Tabs value={'bailo'}>
+                <Tab label={`Models ${models ? `(${models.length})` : ''}`} value='bailo' />
+              </Tabs>
+            </Box>
+            <div data-test='modelListBox'>
+              {(!models || models.length === 0) && <EmptyBlob data-test='emptyModelListBlob' text='No models here' />}
+              {models &&
+                models.map((model: Model, index: number) => {
+                  const latestVersion = model.latestVersion as Version
+                  return (
+                    <Fragment key={model.uuid}>
+                      <Link href={`/model/${model.uuid}`} passHref legacyBehavior>
```

Comment:
We shouldn't be adding legacy behaviour to new pages.

Reply:
Left over from the original page, removed.